packages/user-facing-text/jest.config.js

```diff
@@ -0,0 +1,63 @@
+const { join } = require("path");
+
+module.exports = {
```

Comment:
Couldn't we have a single reusable config for all packages? It would then be a bit easier to maintain it.

Reply:
Will create...